frontend/test/__support__/mocks.js

```diff
@@ -34,3 +36,5 @@ jest.mock("ace/snippets/sql", () => {}, { virtual: true });
 jest.mock("ace/snippets/json", () => {}, { virtual: true });
 jest.mock("ace/snippets/json", () => {}, { virtual: true });
 jest.mock("ace/ext-language_tools", () => {}, { virtual: true });
+
+jest.mock("node-fetch", () => jest.fn());
```

Comment:
This solves the issues with the unit tests epically failing - jest was complaining about `import` being called outside of a module, so [this solution](https://stackoverflow.com/questions/72641681/how-to-use-node-fetch-with-jest-and-typescript) does the trick

Reply:
ah, no it doesn't. it opens up some other errors. Going to see if we can do everything without `node-fetch` here